### PR TITLE
allow media atom maker in DEV to hit CODE

### PR DESCRIPTION
**Pull request by akash1810**
allow media atom maker DEV to access `/api/sections` and `/api/atom/:editorId`

#### Editorial tools integration tests
The editorial tools integration tests live [here](https://circleci.com/gh/guardian/editorial-tools-integration-tests) and get run every time workflow-frontend is deployed to the CODE environment. You should run them before merging this change to master. The current status of the tests (based off the last thing which was deployed to CODE) is:
[CircleCI](https://circleci.com/gh/guardian/editorial-tools-integration-tests)

### CODE REVIEW

**Comment by marialivia16** [comment]
Looks good, but this won't hit CODE because of [this](https://github.com/guardian/workflow-frontend/blob/master/app/config/Config.scala#L17) config.

**Comment by akash1810** [comment]
Updated now, its not the best code ever written, but should be ok for now... I've added a `TODO` 😈

File: app/config/Config.scala [hunk 1]
```diff
@@ -28,6 +32,12 @@ object Config extends AwsInstanceTags {
 
   lazy val mediaAtomMakerUrl: String = s"https://video.$domain"
 
+  // TODO do this better!
+  lazy val mediaAtomMakerUrlForCode: String = stage match {
```
Comment (marialivia16) [inline app/config/Config.scala]: Maybe not the best name as it will handle `PROD` as well, but it'll do for now 👀

Comment (akash1810) [inline app/config/Config.scala]: yeah, I couldn't think of anything better

**Pull request merged**